.github/workflows/build-docker-images.yml

```
@@ -0,0 +1,211 @@
+    if: needs.configure.outputs.has-changes == 'true'
+    runs-on: astro-web3-arc-runner-set
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      fail-fast: false
+      matrix:
+        include: ${{ fromJSON(needs.configure.outputs.matrix) }}
+    steps:
+      - name: Export pod env to GitHub env
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:40: Double quote to prevent globbing and word splitting

(shellcheck)